Registry/Registry.EM_COMMON

```diff
@@ -894,6 +894,20 @@ state    real   SFW1_URB3D      i{umap4}j   misc       1         Z     r      "S
 state    real   SFW2_URB3D      i{umap4}j   misc       1         Z     r      "SFW2_URB3D"  "SENSIBLE HEAT FLUX FROM URBAN SFC"  "W m{-2}"
 state    real   SFR_URB3D       i{umap5}j   misc       1         Z     r      "SFR_URB3D"  "SENSIBLE HEAT FLUX FROM URBAN SFC"  "W m{-2}"
 state    real   SFG_URB3D       i{umap0}j   misc       1         Z     r      "SFG_URB3D"  "SENSIBLE HEAT FLUX FROM URBAN SFC"  "W m{-2}"
+state    real   EP_PV_URB3D     ij          misc       1         -     rh      "EP_PV_URB3D"  "ELEC. PRODUCTION OF ROOFTOP PV PANELS" "W m{-2}" !PVP
```

Comment:
All of these variables need to be in a package

Reply:
Thanks! will add to a package in an updated version.